src/test/java/Homework17.java

```diff
@@ -0,0 +1,38 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class Homework17 extends BaseTest {
+    public void choosePlaylist() {
+
+    }
+
+    public void clickAddBtn() throws InterruptedException {
+        WebElement addToBtn = driver.findElement(By.cssSelector("button.btn-add-to"));
+        addToBtn.click();
+        Thread.sleep(2000);
+    }
+
+    public void selectFirstSongResult() throws InterruptedException {
+        WebElement firstSongResult = driver.findElement(By.cssSelector("section#songResultsWrapper tr.song-item td.title"));
+        firstSongResult.click();
+        Thread.sleep(2000);
+
+    }
+
+    public void clickViewAllBtn() throws InterruptedException {
+        WebElement viewAllSearchResult = driver.findElement(By.cssSelector("div.results section.song h1 button"));
+        viewAllSearchResult.click();
+        Thread.sleep(2000);
+    }
+}
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._

Reply:
Will Do!!!!!